### PR TITLE
several opinionated, cosmetic changes

### PR DESCRIPTION
**Pull request by msagarpatel**
These are several (somewhat opinionated) cosmetic changes. Below is the end result. I expect some of these changes may not be suitable for a wider audience, so let me know if any commits should be dropped from this PR.

Screen Shot 2022-03-04 at 12 28 01 AM

### CODE REVIEW

File: autoload/tagtracks.vim [hunk 1]
```diff
@@ -79,12 +74,22 @@ function tagtracks#StartTagTracks()
   endif
   " window id whose tagstack we want to display
   const tagtracks_id = win_getid()
-  new
+
+  " make HUD 1/4 the height of current window instead of the default 1/2
+  const win_height = winheight(0) / 4
+  execute win_height.'new'
+
```
Comment (benknoble) [inline autoload/tagtracks.vim]: Delete empty line here

File: autoload/tagtracks.vim [hunk 2]
```diff
@@ -79,12 +74,22 @@ function tagtracks#StartTagTracks()
...
+  setlocal nowrap
   setlocal buftype=nofile
   setlocal bufhidden=wipe
   setlocal noswapfile
   setlocal nobuflisted
+
+  " set colors in display to mirror :tags
+  syntax match Title /\vItem\s+Tag\s+Origin/
+  syntax match Directory /\v\d+\s+\S+\s+\S+:\d+\s+\zs.*$/
+
```
Comment (benknoble) [inline autoload/tagtracks.vim]: Let's delete these empty lines too, so the "new window" bit is all grouped together.

**Review by benknoble — CHANGES_REQUESTED** [review]
I'm not too sure about removing the match number. It can be clutter when it isn't needed, but when you have multiple matching tags (happens in languages with overloading/overriding when the tags generator doesn't provide a good disambiguator) it can be nice to know where you are.

**Comment by msagarpatel** [comment]
Yeah, I run into tags with the same identifier all the time, but I don't like seeing the match number after the disambiguation screen. Probably a combination of personal preference, and my workflow. I dropped that commit, and update the others to take the drop into account.

The match number was the one I most thought would not be good in general, so I'll probably just keep that one in my tree.

File: autoload/tagtracks.vim (hunk 1 above)
Comment (msagarpatel) [reply autoload/tagtracks.vim]: Done

File: autoload/tagtracks.vim (hunk 2 above)
Comment (msagarpatel) [reply autoload/tagtracks.vim]: And done

**Review by benknoble — APPROVED** [review]

**Comment by benknoble** [comment]
Works for me. If the desire grows, adding a "smarter" match field or some way to configure it on/off wouldn't be awful, but incurs more complexity.

**Pull request merged**